workflows/diagnostics/fv3net/diagnostics/prognostic_run/emulation/query.py

```diff
@@ -0,0 +1,35 @@
+import wandb
+from dataclasses import dataclass
+
+
+@dataclass
+class PrognosticRunClient:
+    tag: str
+    project: str
+    entity: str
+    api: wandb.Api
+
+    def use_artifact_in(self, run):
+        """Gets prognostic rundir artifact.
+
+        Makes an explicit link the wandb web console.
+        """
+        artifact_name = self.tag + ":latest"
+        return run.use_artifact(artifact_name)
+
+    def _get_run(self):
+        """Return wandb.Run object for this prognostic run"""
+        runs = self.api.runs(
+            filters={"group": self.tag}, path=f"{self.entity}/{self.project}"
+        )
+        prognostic_runs = []
+
+        for run in runs:
+            if run.job_type == "prognostic_run":
+                prognostic_runs.append(run)
+        (run,) = prognostic_runs
```

Comment:
Is there an assumption here that there is only ever one prognostic run for a given tag? 

If you rerun a prognostic run with a model that was used in a previous prognostic run, is a different tag generated? Or do tags correspond to a model configuration?

Reply:
We discussed this on slack. yes there is this assumption.